app/src/main/java/org/thoughtcrime/securesms/groups/compose/CreateGroupScreen.kt

```diff
@@ -0,0 +1,183 @@
+        onGroupNameChanged = viewModel::onGroupNameChanged,
+        groupNameError = viewModel.groupNameError.collectAsState().value,
+        contactSearchQuery = viewModel.selectContactsViewModel.searchQuery.collectAsState().value,
+        onContactSearchQueryChanged = viewModel.selectContactsViewModel::onSearchQueryChanged,
+        onContactItemClicked = viewModel.selectContactsViewModel::onContactItemClicked,
+        showLoading = viewModel.isLoading.collectAsState().value,
+        items = viewModel.selectContactsViewModel.contacts.collectAsState().value,
+        onCreateClicked = viewModel::onCreateClicked,
+        onBack = onBack,
+    )
+}
+
+@OptIn(ExperimentalMaterial3Api::class)
+@Composable
+fun CreateGroup(
+    groupName: String,
+    onGroupNameChanged: (String) -> Unit,
+    groupNameError: String,
+    contactSearchQuery: String,
+    onContactSearchQueryChanged: (String) -> Unit,
+    onContactItemClicked: (accountID: AccountId) -> Unit,
+    showLoading: Boolean,
+    items: List<ContactItem>,
+    onCreateClicked: () -> Unit,
+    onBack: () -> Unit,
+    modifier: Modifier = Modifier
+) {
+    val focusManager = LocalFocusManager.current
+
+    Scaffold(
+        containerColor = LocalColors.current.backgroundSecondary,
+        topBar = {
+            BackAppBar(
+                title = stringResource(id = R.string.groupCreate),
+                backgroundColor = LocalColors.current.backgroundSecondary,
+                onBack = onBack,
+            )
+        }
+    ) { paddings ->
+        Box(modifier = modifier.padding(paddings),) {
+            Column(
+                modifier = modifier.padding(vertical = LocalDimensions.current.spacing),
+                horizontalAlignment = Alignment.CenterHorizontally,
+                verticalArrangement = Arrangement.spacedBy(LocalDimensions.current.spacing)
+            ) {
+                SessionOutlinedTextField(
+                    text = groupName,
+                    onChange = onGroupNameChanged,
+                    placeholder = stringResource(R.string.groupNameEnter),
+                    textStyle = LocalType.current.base,
+                    modifier = Modifier.padding(horizontal = LocalDimensions.current.spacing),
+                    error = groupNameError.takeIf { it.isNotBlank() },
+                    enabled = !showLoading,
+                    innerPadding = PaddingValues(LocalDimensions.current.smallSpacing),
+                    onContinue = focusManager::clearFocus
+                )
+
+                SearchBar(
+                    query = contactSearchQuery,
+                    onValueChanged = onContactSearchQueryChanged,
+                    placeholder = stringResource(R.string.searchContacts),
+                    modifier = Modifier.padding(horizontal = LocalDimensions.current.spacing),
+                    enabled = !showLoading
+                )
+
+                LazyColumn(modifier = Modifier.weight(1f)) {
```

Comment:
```suggestion
                Surface(
                    modifier = Modifier.weight(1f)
                        .nestedScroll(rememberNestedScrollInteropConnection()),
                    color = LocalColors.current.backgroundSecondary
                ) {
                    LazyColumn(modifier = Modifier.fillMaxHeight()) {
                        multiSelectMemberList(
                            contacts = items,
                            onContactItemClicked = onContactItemClicked,
                            enabled = !showLoading
                        )
                    }
                }
```
Because this screen exists  inside a bottom sheet there are conflicts with the scrolling. If you scroll to the bottom of your contacts, swiping down closes the sheets which is not what you want.
We can instead wrap the LazyColumn with a nested scrolling handler

Reply:
Yep